app/src/main/java/org/onionshare/android/tor/MoatApiFactory.kt

```diff
@@ -4,14 +4,14 @@ import org.briarproject.moat.MoatApi
 import java.io.File
 
 fun interface MoatApiFactory {
-    fun createMoatApi(obfs4Executable: File, obfs4Dir: File): MoatApi
+    fun createMoatApi(lyrebirdExecutable: File, lyrebirdDir: File): MoatApi
 }
 
 object DefaultMoatApiFactory : MoatApiFactory {
-    private const val MOAT_URL = "https://onion.azureedge.net/"
-    private const val MOAT_FRONT = "ajax.aspnetcdn.com"
+    private const val MOAT_URL = "https://1314488750.rsc.cdn77.org/"
+    private const val MOAT_FRONT = "www.phpmyadmin.net"
```

Comment:
Not necessarily a blocker, given that the Azure front is about to disappear for all users, but the CDN77 front won't work for devices running Android < 7.1.1 because they can't verify its Let's Encrypt cert. I've emailed the Tor devs to ask whether a solution is in the pipeline or whether we should skip Moat on those devices for now.

Reply:
Good to know, thanks. Do you know when the azure front will go away? Late enough to justify keeping it for now?

Reply:
Apparently it could happen as early as 15 Jan. The story seems to be that Edgio went bankrupt and was acquired, so it's up to the new owner whether to allow a longer transition period.

https://learn.microsoft.com/en-us/azure/cdn/edgio-retirement-faq

Reply:
The Tor devs replied that Tor's initial Moat connection goes via Lyrebird, which is being (or has already been?) patched to accept the Let's Encrypt certs. So I guess we should update moat-api to do the same.

Reply:
great news. 

> So I guess we should update moat-api to do the same.

Do we merge this here in the meantime and update moat-api separately before next release?

Reply:
I took another look at the code and it seems we're already making the initial connection via Lyrebird. I forgot that we already added this because the domain-fronted Moat servers communicate via meek, not direct TLS connections. So we just need to check whether Lyrebird correctly handles the CDN77 cert on an old device.

BTW I used the wrong URL for CDN77 in moat-api - I've just opened an MR to update it, and it looks like it might need to be updated here too.

Reply:
is it correct now?